--- dev.config.js.orig	2016-08-06 10:18:57.503347424 +0300
+++ dev.config.js	2016-08-06 10:17:53.967347424 +0300
@@ -5,9 +5,9 @@
 import baseConfig from './base.config';
 import startKoa from './utils/start-koa';
 
-const LOCAL_IP = require('dev-ip')();
+// const LOCAL_IP = require('dev-ip')();
 const PORT = parseInt(process.env.PORT, 10) + 1 || 3001;
-const HOST = (isArray(LOCAL_IP) && LOCAL_IP[0]) || LOCAL_IP || 'localhost';
+const HOST = '127.0.0.1'; // (isArray(LOCAL_IP) && LOCAL_IP[0]) || LOCAL_IP || 'localhost';
 const PUBLIC_PATH = `//${HOST}:${PORT}/assets/`;
 
 export default {
